src/server/page.html.jsx

```diff
@@ -23,6 +23,12 @@ globalStyles.cssRule('body', {
   fontFamily: 'Verdana, Geneva, sans-serif'
 })
 
+globalStyles.cssRule('@media all and (display-mode: standalone)', {
```

Comment:
I find it strange that this is a CSS rule - but ok, why not - keeps it simple - works in my tests